### PR TITLE
Product listing

### PR DESCRIPTION
**Pull request by AnuragSahu11**
*Added product listing page
https://deploy-preview-4--wizardly-torvalds-416226.netlify.app/pages/productlisting

### CODE REVIEW

File: pages/productListing.html [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+                    <i></i>
+                    <span class="form-radio-button-text is-2 is-medium">1 Star & above</span>
+                  </label></li>
+            </ul>
+        </div>
+        <div class="list-container">
+            <div class="title semibold">Sort By</div>
+            <ol class="list list-none">
+                <li class="list-items"><label class="form-radio-label">
+                    <input type="radio" name="opinion" checked />
```
Comment (RakshithBhat03) [inline pages/productListing.html:114]: You can change the value of the name attribute here. "Sort by" and "Rating" should have different radio names.

**Review by RakshithBhat03 — COMMENTED** [review]
Everything else looks good

**Review by Hamdrive — COMMENTED** [review]
Looks good but you want to look into your filters as they seem to be breaking and overflowing

image

File: pages/productListing.html (hunk 1 above)
Comment (AnuragSahu11) [reply pages/productListing.html:114]: Yes I will see to that

**Pull request merged**